GliaWidgets/Sources/View/Chat/UnreadMessageDividerView.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 import UIKit
 
 final class UnreadMessageDividerView: BaseView {
+    static let accessibilityIdentifier = "unread_message_divider_accessibility_identifier"
```

Comment:
For the previous accessibility identifiers, we would use something like functionality_description_typeofview. So for example here it could have been chatTranscript_unreadMessageDivider_view. Other parts have it like chat_transcript_unread_message_divider_view. Kind of sucks that we don't have a standard, but I believe that we should at least have the format I mentioned.

Reply:
Good point d9809248b6e12bb627f0ed19f522d238a362033d.